internal/server/build.ts

```
@@ -117,7 +115,12 @@ class Builder {
     if (this.inited) return this;
     this.Check_remove_node_modules_files_path();
     await this.InitGetCustomPluginsFromUser();
-    await this.InitAfterBuildPlugins();
+    this.remove_node_modules_files_path.push(
+      ...(await this.getPlugins())
+        .map((p) => p.removeFromBuild)
+        .filter((p) => p != undefined)
+        .reduce((p, n) => [...p, ...n], [])
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)